### DISCUSSION TITLE
Synchronize between Object Storages in different regions

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#3131] [Q&A]
Command:

```
juicefs sync 
  s3://xxx:xxx@bucket.s3.sbg.io.cloud.ovh.net
  s3://xxx:xxx@bucket.s3.gra.io.cloud.ovh.net
```

I need to pass a different region for each Object Storage (`AWS_REGION` env var - `sbg` and `gra`) how can i do that?
Since there is no `--region` param this seems complicated to set it twice... https://github.com/juicedata/juicefs/issues/3129#issuecomment-1366540293

**Comment by zhijian-pro** [comment]
That's a good question, and we're going to implement automatic region resolution support for OVH.

**Reply by solracsf** [reply]
This doesn't apply only to OVH.
